apps/_compability_app/package.json

```diff
@@ -34,11 +34,18 @@
     "get-contrast": "^2.0.0",
     "graphql": "^15.3.0",
     "graphql-tag": "^2.11.0",
+    "immutable": "^4.0.0-rc.12",
+    "inversify": "^5.0.1",
     "lodash": "^4.17.20",
     "next": "^9.5.2",
+    "next-images": "^1.4.1",
     "react": "^16.13.1",
+    "react-chat-elements": "^10.10.2",
     "react-dom": "^16.13.1",
-    "react-intl": "^5.7.1"
+    "react-intl": "^5.7.1",
```

Comment:
it's better to use it from `@core/next/intl`. It prevents to import two different versions in the same project